spring-session/src/main/java/org/springframework/session/data/couchbase/Serializer.java

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
I don't think we should have a couchbase specific API here. We could use something like Spring's converter APIs to convert the values.